pkg/network/tracer/tracer_linux_test.go

```diff
@@ -2016,65 +2014,6 @@ func (s *TracerSuite) TestPreexistingConnectionDirection() {
 	assert.Equal(t, network.INCOMING, incoming.Direction)
 }
 
-func (s *TracerSuite) TestPreexistingEmptyIncomingConnectionDirection() {
```

Comment:
Can we add a test instead to verify if we are dropping these kinds of connections?